app/__main__.py

```diff
@@ -0,0 +1,25 @@
+"""Full Translation App"""
+
+from os import getenv
+from pyrogram import Client
+from consumers import TelegramConsumer
+from translators import PythonTranslator
+from app import TranslationApp
+
+api_id = getenv('API_ID')
+api_id_hash = getenv('API_ID_HASH')
+# TODO(walt): consider whether to move this construction into app.py, and if so how
+client = Client("my_account", api_id, api_id_hash)
+
+app = TranslationApp(client, 'me') # Replace 'me' with -1001173826177 for real HK channel

```

Comment:
maybe use named parameters here for chat_id to be more explicit

Reply:
Good call